packages/app-store/office365calendar/lib/CalendarService.ts

```
@@ -619,10 +674,402 @@ export default class Office365CalendarService implements Calendar {
+        items,
+        // Expand the start date to the start of the month to increase cache hits
+        timeMin: getTimeMin(timeMin),
+        // Expand the end date to the end of the month to increase cache hits
+        timeMax: getTimeMax(timeMax),
+      },
+      // For delegation scenarios, use userId instead of credentialId
+      credentialId: this.credential.id,
+      userId: this.credential.delegatedTo ? this.credential.userId : null,
+    });
+
+    if (cachedAvailability && cachedAvailability.value) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)